pypi_formula_mappings.json

```diff
@@ -64,6 +64,9 @@
   "aws-sso-util": {
     "exclude_packages": ["certifi"]
   },
+  "awscli": {
+    "extra_packages": ["setuptools", "wheel"]
+  },
```

Comment:
Added these so that `update-python-resources` doesn't remove them. They're still needed so that `awscrt` will have them.